doc/user/content/get-started.md

```diff
@@ -5,216 +5,271 @@ menu: "main"
-            (val->'revision'->'old')::float::int AS revision_old,
-            val->>'server_name' AS server_name,
-            (val->'server_script_path')::text AS server_script_path,
-            val->>'server_url' AS server_url,
-            (val->'timestamp')::float AS r_ts,
-            val->>'title' AS title,
-            val->>'type' AS type,
-            val->>'user' AS user,
-            val->>'wiki' AS wiki
-        FROM (SELECT data::jsonb AS val FROM wikirecent);
-    ```
-
-1. From here we can start building our aggregations. The simplest place to start is simply counting the number of items we've seen:
+Joins are one of the most powerful features in SQL, but something quite hard to get right in the streaming world. Materialize efficiently supports [all types of SQL joins](/sql/join/#examples) under all the conditions you would expect from a traditional relational database.
+
+1. Create and populate a table with static, reference data:
 
     ```sql
-    CREATE MATERIALIZED VIEW counter AS
-        SELECT COUNT(*) FROM recentchanges;
+    CREATE TABLE symbols (
+        symbol text,
+        ticker text
+    );
+
+    INSERT INTO symbols
+    SELECT * 
+    FROM (VALUES ('Apple','AAPL'), 
+                 ('Google','GOOG'), 
+                 ('Elerium','ELER'), 
+                 ('Bespin Gas','BGAS'),
+                 ('Linen Cloth','LCLO')
+    );
+
     ```
 
-1. However,  we can also see more interesting things from our stream. For instance, who are making the most changes to Wikipedia?
+    **Note:** We are using a table for convenience, to avoid adding complexity to the guide. It's [unlikely](/sql/create-table/#when-to-use-a-table) that you'll need to use tables in real-world scenarios.
+
+1. Now, we can enrich our aggregated data with the ticker for each stock using a regular `JOIN`:
 
     ```sql
-    CREATE MATERIALIZED VIEW useredits AS
-        SELECT user, count(*) FROM recentchanges GROUP BY user;
+    CREATE MATERIALIZED VIEW cnt_ticker AS
+    SELECT s.ticker AS ticker, 
+           COUNT(*) AS cnt
+    FROM market_orders m
+    JOIN symbols s ON m.symbol = s.symbol
+    GROUP BY s.ticker;
     ```
 
+    To see the results:
+
     ```sql
-    SELECT * FROM useredits ORDER BY count DESC;
+    SELECT * FROM cnt_ticker;
+     ticker | cnt
+    --------+-----
+     AAPL   |  42
+     BGAS   |  49
+     ELER   |  68
+     GOOG   |  51
+     LCLO   |  70
     ```
+{{< /expand >}}
 
-1. If this is a factoid we often want to know, we could also create a view of just the top 10 editors we've seen.
+{{< expand "Temporal Filters" >}}
+
+A common requirement in streaming use cases is time-windowing, for example to model business processes or simply to bound resource usage. In Materialize, you can express this pattern using [temporal filters](/guides/temporal-filters/).
+
+1. If instead of computing and maintaining the _overall_ count we want to get the _moving_ count over the past minute:
 
```

Comment:
Commas after "If" and the first "count", although actually I think we can cut the phrase altogether. We should probably be a little more descriptive in the step:

"To get the count of stock orders made over the last minute, instead of setting a static time, we create a materialized view using a temporal filter that will always compute results from *now*, whenever *now" happens to be:

Reply:
You're absolutely right, this could be a bit more helpful. I wanted to avoid going into the whole explaining `mz_logical_timestamp()`, but it does lack context as is.